index.html

```diff
@@ -104,15 +104,15 @@ <h3 class="article__title">How can I help?</h3>
             </div>
           </div>
         </div>
-
-        <div class="sidebar">
+ 
+        <aside class="sidebar">
           <h2 class="heading-underline">Upcoming events</h2>
 
           <div class="article">
```

Comment:
This can be changed to `article` tag as mentioned above and add more spacing please around the texts.